.circleci/config.yml

```diff
@@ -0,0 +1,13 @@
+
+version: 2
+jobs:
+    build:
+        docker:
+            - image: theorangeone/docker-pandoc
+        working_directory: ~/labs/electronics/electronics
```

Comment:
I think this is the working directory for the whole run, so the code is checked out as a subdirectory _of this folder_. I suspect that we need to add a `cd electronics` to the `make` line.

An even better fix might be to have a top level `Makefile` which makes all the children, though that may also be a bit more involved to make work nicely.